### PR TITLE
Merging Development to Master for Deployment

### PR DESCRIPTION
**Pull request by leekahung**
Unsure how to rebase Master/Development for merging Development to Master.

### CODE REVIEW

**Comment by leekahung** [comment]
Turning this into a draft for the time being, until Master can be rebased properly

File: src/components/Login/Logout.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ * @memberof Login
+ * @component
+ * @name Logout
+ */
+
+const Logout = () => {
+  const { session } = useSession();
+
+  return (
+    <section id="login" className="panel">
```
Comment (leekahung) [inline src/components/Login/Logout.jsx]: id="login" is incorrect, should be id="logout"

**Review by leekahung — COMMENTED** [review]
Need to fix one of the HTML id's; can't merge into Master until it's fixed in Development

File: src/components/Login/Logout.jsx (hunk 1 above)
Comment (leekahung) [reply src/components/Login/Logout.jsx]: Issue resolved in #52.

**Comment by leekahung** [comment]
I think everything has been properly rebased. Switching from draft to allow for reviews. Summary of recent changes in Development includes:

- Updates to GitHub Actions workflow with ci.yml instead of build.yml
- New functionality to generate ACL file for Solid containers
- New functionality to set ACL permissions
- Linting/Formatting from ESLint and Prettier
- Refactoring of components like Login into Login/Logout and StatusNotification into StatusMessage/StatusNotification, etc.
- Moving SessionProvider up one more level from App.jsx to index.jsx
- Refactoring session.info.isLoggedIn from other components to just App.jsx
- Reorganizing all Form components to end with Form
- Included new directory to Notification components (StatusMessage, and StatusNotification)
- Split session-helper.js into session-core.js and session-helper.js
- Updates to JSDoc comments

**Review by Jared-Krajewski — APPROVED** [review]
checked on local and looks good. Maybe see if we can get at least a second set of eyes on it and do the merge.

**Pull request merged**